Makefile

```diff
@@ -0,0 +1,45 @@
+NAME=cimgui-go Code Generator
+
+.PHONY: all
+
+## all: generates both bindings.
+all: generate
+
+.PHONY: setup
+## setup: pulls down dependencies.
+setup:
```

Comment:
Do you actually need this? "go run" pulls dependencies automatically.

Reply:
> and maybe in th efuture, if we'll need to use go generate or something like that, we may put it there.

also, I'm wondering if we shouldn't obtain gofmt here? User isn't obligated to have it installed tbh.

Reply:
> and maybe in the future, if we'll need to use go generate, stringer or something like that, we may put it there.
I don't like leaving unnecessary things "for the future" - we can easily add the targets we need later. It will be cleaner.

> also, I'm wondering if we shouldn't obtain gofmt here?
Go ships with gofmt, so this is not needed.

Reply:
okey, sounds fair